corehq/apps/case_importer/static/case_importer/js/import_history.js

```diff
@@ -13,13 +13,33 @@ hqDefine('case_importer/js/import_history.js', function () {
         };
         self.case_uploads = ko.observableArray(null);
         self.state = ko.observable(self.states.NOT_STARTED);
-        var shouldUpdate = function (data) {
-            // do not update DOM
-            // if we're not either adding new uploads or updating the status
-            // this prevents some jumpiness when not necessary
-            // and is particularly bad if you're in the middle of editing a comment
-            return !(_.chain(self.case_uploads()).pluck('upload_id').isEqual(_(data).pluck('upload_id')).value() &&
-                _.chain(self.case_uploads()).pluck('task_status').isEqual(_(data).pluck('task_status')).value());
+        var uploadIdsInDataMatchCurrent = function (data) {
+            return _.chain(self.case_uploads()).pluck('upload_id').isEqual(_(data).pluck('upload_id')).value();
+        };
+        var taskStatusesInDataMatchCurrent = function (data) {
+            return (
+                _.chain(self.case_uploads()).pluck('task_status').map(function (task_status) {return task_status()})
+                            .isEqual(_(data).pluck('task_status').map(function (task_status) {return task_status()})).value()
```

Comment:
oh i get it. it's complaining about the missing semicolon after `return task_status()`. Tricky. Fixed.